<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="css/index.css">
<link rel="stylesheet" type="text/css" href="css/ProjectOverview.css">
<script type="text/javascript" src="./js/jquery-1.11.1.min.js"></script>
<script type="text/javascript" src="./js/nav.js"></script>
<title>Protocols</title>
</head>

<body>
<div class="content-container">
  <div class="main-container">
    <div class="content">
    
    
      <div id="protocol-12" class="region">
        <div id="protocol-1" class="sub-region shade">
          <div class="h0title clear">1. Buffers/Medium</div>
          <div class="h2white clear">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;LB medium</div>
          <p class="clear">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Sodium Chloride 10 g/L<br/>
            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Tryptone 10 g/L<br/>
            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Yeast extract 5 g/L</p>
          <div class="h2white clear">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;For agar</div>
          <p class="clear">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;BacAgar (1%) 15 g/L</p>
        </div>
        <div id="protocol-2" class="sub-region shade">
          <div class="h0title clear">2. Add Restriction Enzyme Cutting Site</div>
          <p class="">In the process of standardization, we need connect the modified fragment to the Connection of EcoRI, XbaI , SpeI, PstpS and B1C3. So we add this four Restriction Enzyme cutting site at the both ends of the modified fragment. The sequence are gaattcgcggccgcttctaga（5'→3'）=ecori-xbai, actagtagcggccgctgcag（5'→3'）=spei-psti, It make standardization be possible. EcoRI and Xbal are located upstream, SpeI and PstI are located downstream.</p>
        </div>
      </div>
      
      
      <div id="protocol-3" class="region shade">
        <div class="h0title clear">3. Enzyme Cut and Ligation</div>
        <div class="sub-region">
          <div class="h2white clear">Enzyme cutting reaction </div>
          <table class="reaction">
            <tr>
              <td class="td_item">10xH Buffer</td>
              <td class="td_val">5μL</td>
            </tr>
            <tr>
              <td class="td_item">0.1%BSA</td>
              <td class="td_val">5μL</td>
            </tr>
            <tr>
              <td class="td_item">DNA</td>
              <td class="td_val">&le;2500ng</td>
            </tr>
            <tr>
              <td class="td_item">Bgl II</td>
              <td class="td_val">2μL</td>
            </tr>
            <tr>
              <td class="td_item">Not I</td>
              <td class="td_val">2μL</td>
            </tr>
            <tr>
              <td class="td_item">aquae sterilisata</td>
              <td class="td_val">Up to 50μL</td>
            </tr>
            <tr>
              <td class="td_item">total</td>
              <td class="td_val">50μL</td>
            </tr>
          </table>
        </div>
        <div id="ligation" class="sub-region">
          <div class="h2white clear">ligation reaction </div>
          <table class="reaction">
            <tr>
              <td class="td_item">T4 Buffer</td>
              <td class="td_val">1μL</td>
            </tr>
            <tr>
              <td class="td_item">T4 ligase</td>
              <td class="td_val">0.5μL</td>
            </tr>
            <tr>
              <td class="td_item">element</td>
              <td class="td_val">1μL</td>
            </tr>
            <tr>
              <td class="td_item">carrier</td>
              <td class="td_val">3μL</td>
            </tr>
            <tr>
              <td class="td_item">ddH2O</td>
              <td class="td_val">4.5μL</td>
            </tr>
            <tr>
              <td class="td_item">total</td>
              <td class="td_val">10μL</td>
            </tr>
          </table>
          <p class="clear float_r">(overnight at 22 ℃)</p>
        </div>
      </div>
      
      
      <div id="protocol-4" class="region shade">
        <div class="h0title clear">4. Add Restriction Enzyme Cutting Site</div>
        <div class="h2white clear">Experimental Material:</div>
        <p class="clear">&nbsp;1.5ml centrifuge tube, competent cells, LB Medium without penicillin.</p>
        <p class="clear" style="line-height:3px;">&nbsp;</p>
        <div class="h2white clear">Steps:</div>
        <p class="clear">I.&nbsp;&nbsp;Prepare the ice, the competent cells and mark the 1.5ml centrifuge tube.<br/>
          II.&nbsp;&nbsp;When the competent cells  dissolve in the ice, inject 50-60μL competent cells and 10μL ligation product into every centrifuge tube, put them in the ice for 30min.<br/>
          III.&nbsp;&nbsp;Water bath at 42℃ for 90s, then in the ice for 3-5min.<br/>
          IV.&nbsp;&nbsp;Inject 500μL LB fluid medium(without antibiotics) into the tube, blending and cultivate in the table concentrator at 37℃,120-150rpm for 45-60min.(when the bacteria returned to normal growth condition and expression the antibiotic resistance genes encoded by plasmid)</p>
      </div>
      
      
      <div id="protocol-5" class="region shade">
        <div class="h0title clear">5. Screening and Identification</div>
        <div class="h2white clear">Experimental Material:</div>
        <p class="clear">Sterilization toothpick, Medium plate with AMP, sealing film, mark pen.</p>
        <p class="clear" style="line-height:3px;">&nbsp;</p>
        <div class="h2white clear">Steps:</div>
        <p class="clear">I.&nbsp;&nbsp;Prepare the medium plate: When the autoclaving LB medium cooling to 50℃（protect the  antibiotic from inactivation）, add 1/1000 antibiotic, blending, pour into the plate fast. When the medium freeze, seal it with sealing film.<br/>
          II.&nbsp;&nbsp;Centrifuge the tube taken out from table concentrator at 4000rpm for 3min. Take 500μL supernatant, put the residuum in the super clean bench. Make it well-distributed with the pipettor. inhale and coating it(50μL) on the plate with corresponding resistance. When the liquid is absorbed by the plate entirely, place upside down the petri dish. Cultivate at 37 ℃ for 12-16h. Save the residuum at 4℃ for later use.<br/>
          III.&nbsp;&nbsp;Put the sterilization toothpick in the super cle an bench, turn on the UV lamp, after 20min, turn off the UV lamp.<br/>
          IV.&nbsp;&nbsp;Light the alcohol lamp, put the LB medium plate and the petri dish after transformation into the super clean bench.<br/>
          V.&nbsp;&nbsp;Find some single colony on the plate and stick them with the toothpick. Then put the toothpicks into the centrifuge tubes which have added the system for PCR. Turn off the centrifuge tubes, place them for some minutes and take out the toothpicks.<br/>
          VI.&nbsp;&nbsp;Save the petri dish at 4℃. Clean the super clean bench and turn off the switch.</p>
          <p class="clear" style="line-height:3px;">&nbsp;</p>
        <div class="h2white clear">PCR reaction：</div>
        <div class="sub-region">
          <table class="reaction">
            <tr>
              <td class="td_item">dNTP Mix</td>
              <td class="td_val">5μL</td>
            </tr>
            <tr>
              <td class="td_item">Primer F</td>
              <td class="td_val">0.2μL</td>
            </tr>
            <tr>
              <td class="td_item">Primer R</td>
              <td class="td_val">0.2μL</td>
            </tr>
            <tr>
              <td class="td_item">ddH2O</td>
              <td class="td_val">4.6μL</td>
            </tr>
            <tr>
              <td class="td_item">Total Volume</td>
              <td class="td_val">10μL</td>
            </tr>
          </table>
        </div>
        
        <div id="pcr" class="sub-region">
        <table class="none_border">
        	<tr>
              <td>95 ℃</td>
              <td>5 min</td>
            </tr>
            <tr>
              <td>95 ℃</td>
              <td>30 s</td>
            </tr>
            <tr>
              <td>60 ℃</td>
              <td>30 s</td>
            </tr>
            <tr>
              <td>72 ℃</td>
              <td>30 s</td>
            </tr>
            <tr>
              <td>72 ℃</td>
              <td>5 min</td>
            </tr>
            <tr>
              <td>16 ℃</td>
              <td>∞</td>
            </tr>
        </table>
        <div class="vertical_align">
        	<img class="parenthesis" src="resources/parenthesis.png" />
        	<p class="float_l">35 Cycles</p>
        </div>
        </div>
      </div>
      
      
      <div id="protocol-6" class="region shade">
        <div class="h0title clear">6. Cultivate the Bacterium and Get the Plasmid</div>
        <p class="clear">I.&nbsp;&nbsp;chose the bacterium in the plate to get the single colony. Streaking in the super clean bench. Don’t rotate the spear and toothpicks when streaking.<br/>
II.&nbsp;&nbsp;inoculate. Need to use the super clean bench.<br/>
III.&nbsp;&nbsp;Pour the fluid medium into Sterilization tube when the liquid level arrive to 1/4. Add single colony into tube and put the tube in the table concentrator at 37℃，220rpm for a night.<br/>
IV.&nbsp;&nbsp;Save the bacterium. Pour 750ml bacterium suspension and 250ml sterilization glycerol into the tube. There are13-1 and 14-16. One is in refrigerator at -20℃ and the other is at -80℃.<br/>
V.&nbsp;&nbsp;Take 1ml bacterium suspension to sequencing.<br/>
VI.&nbsp;&nbsp;plasmid extraction: use the High purity plasmids small kits of TianGen</p>
      </div>
      
      
      <div id="protocol-7" class="region shade">
        <div class="h0title clear">7. Detection</div>
        <p class="clear">Detect the segment inside of Restriction Enzyme cutting site and identify the effectiveness of standardization.</p>
      </div>
      
      
      <div id="protocol-8" class="region shade">
        <div class="h0title clear">8. Recycle Fragment using KOD Enzyme</div>
        <div class="h2white">PCR reaction</div>
        <div class="sub-region clear">
         <table class="reaction">
            <tr>
              <td class="td_item">Sense primer</td>
              <td class="td_val">1μL</td>
            </tr>
            <tr>
              <td class="td_item">reverse primer</td>
              <td class="td_val">1μL</td>
            </tr>
            <tr>
              <td class="td_item">template DNA</td>
              <td class="td_val">1μL</td>
            </tr>
            <tr>
              <td class="td_item">2xKOD Buffer</td>
              <td class="td_val">25μL</td>
            </tr>
            <tr>
              <td class="td_item">KOD</td>
              <td class="td_val">1μL</td>
            </tr>
            <tr>
              <td class="td_item">dNTPs</td>
              <td class="td_val">10μL</td>
            </tr>
            <tr>
              <td class="td_item">ddH20</td>
              <td class="td_val">11μL</td>
            </tr>
            <tr>
              <td class="td_item">Total Volume</td>
              <td class="td_val">50μL</td>
            </tr>
          </table>
          </div>
          <div class="sub-region">
          <table class="none_border">
        	<tr>
              <td>94 ℃</td>
              <td>3 min</td>
            </tr>
            <tr>
              <td>98 ℃</td>
              <td>30 s</td>
            </tr>
            <tr>
              <td>55 ℃</td>
              <td>10 s</td>
            </tr>
            <tr>
              <td>72 ℃</td>
              <td>70 s</td>
            </tr>
            <tr>
              <td>16 ℃</td>
              <td>∞</td>
            </tr>
        </table>
          <div class="vertical_align">
        	<img class="parenthesis" src="resources/parenthesis.png" />
        	<p class="float_l">30 Cycles</p>
        	</div>
        </div>
        <p class="clear" style="line-height:3px;">&nbsp;</p>
        <div class="h2white clear">Plasmid linearization reaction</div>
      	<div class="sub-region">
        <table class="reaction">
            <tr>
              <td class="td_item">10xH Buffer</td>
              <td class="td_val">5μL</td>
            </tr>
            <tr>
              <td class="td_item">DNA</td>
              <td class="td_val">&le;2500ng</td>
            </tr>
            <tr>
              <td class="td_item">Hind III</td>
              <td class="td_val">2μL</td>
            </tr>
            <tr>
              <td class="td_item">aquae sterilisata</td>
              <td class="td_val">Up to 50μL</td>
            </tr>
            <tr>
              <td class="td_item">Total Volume</td>
              <td class="td_val">50μL</td>
            </tr>
          </table>
        </div>
      <div class="sub-region">
      <p id="plasmid" class="clear">water bath at 37℃ for 3 h. purify the product using E.Z.N.AR  Cycle-Pure Kit (OMEGA) after Single enzyme digestion.</p>
      </div>
      </div>
      
      
      <div id="protocol-9" class="region shade">
        <div class="h0title clear">9. Transformation of Pichia pastoris Competent Cell</div>
        <p class="clear">I.&nbsp;&nbsp;add 0.1-0.2μgDNA into every 80μL pichia pastoris competent cell, move it to 0.2 cm electric shock cup, ice-bath for 5 min;<br/>
II.&nbsp;&nbsp;electric shock the competent cell at 1.5kV, 200Ω, 25mF, 5ms;<br/>
III.&nbsp;&nbsp;add 1mL Cold sorbitol after electric shock, move it to 1.5mL centrifuge tube, put it on a stable plane for 1.5 h.<br/>
IV.&nbsp;&nbsp;lay over 200μL bacterium solution on YPDSZ plate (with resistance of  Zeocin, 100μg/mL), cultivate at 30℃ for 3 days. Observe the result.
</p>
      </div>
      

	<div id="protocol-10" class="region shade">
        <div class="h0title clear">10. Screening of Positive Transformation of Recombinant Pichia pastoris </div>
        <p class="clear">I.&nbsp;&nbsp;chose 10 transformants from every YPDSZ plate. Mark them and inoculate at YPD plate for storage. At the same time, lay over it at the PCR tubes, try to make the bacterium adhere to the bottom of the tube;<br/>
II.&nbsp;&nbsp;put the sample and a 200mL beaker with 50mL water in the microwave oven, set the MED-LOW fire and wait for 5 min;<br/>
III.&nbsp;&nbsp;centrifuge at 6000rpm for 1min. Test using PCR(10μL);<br/>
IV.&nbsp;&nbsp;electrophoresis detection of the PCR product in 1% sepharose gel;<br/>
V.&nbsp;&nbsp;knockout the box1: Up-lox71-AOX-Cre-Zeocin-lox66-Down, transform to Gs115/Gcw61-CALB using electric shock. Screening transformant of homologous recombination using YPDSZ plate.<br/>
VI.&nbsp;&nbsp;use the P1, P6 primer. Identify the positive transformant by PCR.<br/></p>

	<div class="h2white">PCR reaction :</div>
	<p class="clear" style="line-height:3px;">&nbsp;</p>
	<p class="clear">VII.&nbsp;&nbsp;use the P7, Pc primer. Identify the positive transformant by PCR. PCR reaction is :</p>
		<table class="reaction clear">
            <tr>
              <td class="td_item">Sense primer</td>
              <td class="td_val">0.8μL</td>
            </tr>
            <tr>
              <td class="td_item">reverse primer</td>
              <td class="td_val">0.8μL</td>
            </tr>
            <tr>
              <td class="td_item">2xKOD Buffer</td>
              <td class="td_val">10μL</td>
            </tr>
            <tr>
              <td class="td_item">KOD</td>
              <td class="td_val">0.4μL</td>
            </tr>
            <tr>
              <td class="td_item">dNTPs</td>
              <td class="td_val">4μL</td>
            </tr>
            <tr>
              <td class="td_item">ddH20</td>
              <td class="td_val">4μL</td>
            </tr>
            <tr>
              <td class="td_item">Total Volume</td>
              <td class="td_val">20μL</td>
            </tr>
          </table>
	<p class="clear">VIII.&nbsp;&nbsp;set the program of PCR:</p>
    <p class="clear" style="line-height:3px;">&nbsp;</p>
		<div class="sub-region clear">
          <table class="none_border">
        	<tr>
              <td>94 ℃</td>
              <td>5 min</td>
            </tr>
            <tr>
              <td>94 ℃</td>
              <td>10 s</td>
            </tr>
            <tr>
              <td>55 ℃</td>
              <td>10 s</td>
            </tr>
            <tr>
              <td>72 ℃</td>
              <td>5 min</td>
            </tr>
             <tr>
              <td>72 ℃</td>
              <td>10 min</td>
            </tr>
            <tr>
              <td>16 ℃</td>
              <td>∞</td>
            </tr>
        </table>
          <div class="vertical_align">
        	<img class="parenthesis" src="resources/parenthesis.png" />
        	<p class="float_l">30 Cycles</p>
        	</div>
        </div>
      </div>

      
    </div>
  </div>
</div>
</body>
</html>

<!--
10. Screening of Positive Transformation of Recombinant Pichia pastoris 
I.     
II.    
III.   
IV.   
V.    knockout the box1: Up-lox71-AOX-Cre-Zeocin-lox66-Down, transform to Gs115/Gcw61-CALB using electric shock. Screening transformant of homologous recombination using YPDSZ plate.
VI.   use the P1, P6 primer. Identify the positive transformant by PCR.
PCR reaction :
VII.	use the P7, Pc primer. Identify the positive transformant by PCR. PCR reaction is :








VIII.	set the program of PCR:

            94℃     5 min
            94℃     10 s
            55℃      10 s                         30 Cycles
            72℃     5 min
            72℃     10 min
            16℃	       ∞



           
 -->